vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -982,15 +987,16 @@ def get_config_dtype_str(
 
 # TODO (bnell): use scalar_type instead of bools?
 def get_config_qtype(
-    use_fp8_w8a8: bool,
-    use_int8_w8a8: bool,
-    use_int8_w8a16: bool,
-    use_int4_w4a16: bool,
-) -> Optional[torch.dtype]:
+        use_fp8_w8a8: bool, use_int8_w8a8: bool, use_int8_w8a16: bool,
+        use_int4_w4a16: bool,
+        use_mxfp4_w4a4: bool) -> Optional[Union[torch.dtype, str]]:
     if use_fp8_w8a8:
         return torch.float8_e4m3fn
     elif use_int8_w8a8:
         return torch.int8
+    elif use_mxfp4_w4a4:
+        # TODO: replace by a torch.dtype once PyTorch natively supports mxfp4.
+        return "mxfp4"
```

Comment:
How is this qtype used? It seems potentially dangerous to not use a torch.dtype here